_rules/SC2-4-4+2-4-9+4-1-2-link-has-name.md

```diff
@@ -125,11 +125,27 @@ When `link` is off screen.
     }
   </style>
   <body>
-		<a class="offScreenLink" href="http://www.w3.org/WAI"> Web Accessibility Initiative (WAI) </a>
+		<a class="offScreenLink" href="http://www.w3.org/WAI">Web Accessibility Initiative (WAI)</a>
   </body>
 </html>
 ```
 
+#### Passed example 10
+
+`area` element with `href` attribute has accessible name.
+
+```html
+<area shape="rect" coords="0,0,82,126" href="http://www.w3.org/WAI" alt="Web Accessibility Initiative (WAI)">
```

Comment:
This shouldn't be applicable. An `area` element isn't focusable unless it has an image associated with it. We should maybe exclude area from this, since it should be applicable to 1.1.1.

Reply:
@WilcoFiers, The rule is applicable to "any HTML element with the semantic role of link that is included in the accessibility tree." It doesn't have to be focusable. 
Since it's about accessible names, it makes sense to align the rule with what is presented to a screen reader. 
And according to https://www.w3.org/TR/html-aria/ an "area with a href" has the implicit ARIA semantics of role=link.

I would say that something that is an non-text content but has the role of a link should fall under both 1.1.1 and 2.4.4/2.4.9.

Reply:
@annethyme This `area` element isn't included in the accessibility tree.

Reply:
If I just use arrow navigation in VoiceOver I don't have any trouble getting to this element. It is read out as a link with the alt text as it's accessible name. Which is in line with the ARIA in HTML spec that says that the area has the role of link - and not only if it has an associated image.
I vote for keeping it this test case as is.

Reply:
I'm seeing the same thing as Anne and cannot find anything in neither the Core nor HTML AAM that would cause an `<area>` element with an `href` attribute to _not_ be included in the accessibility tree.
<img width="697" alt="image" src="https://user-images.githubusercontent.com/1174718/57073300-f3c8ad80-6ce0-11e9-8c5b-b80f9423634c.png">